app/controllers/collections_controller.rb

```diff
@@ -121,6 +121,93 @@ def revoke_access
     redirect_to licences_path
   end
 
+  def add_items_to_collection
+    # referenced documents (HCSVLAB-1019) are already handled by the look_for_documents part of the item ingest
+    collection = Collection.find_by_name(params[:id])
+    if !request.post? or params[:items].blank?
+      respond_with_error("JSON-LD formatted item metadata must be sent to the api call as a POST request", 400)
```

Comment:
You can limit the types of requests that will be accepted in the `config/routes.rb`, so you won't need to check if it's a POST or not